src/server.js

```diff
@@ -22,9 +22,19 @@ const supportedFormats = [
     'jpg',
     'pdf',
     'png',
-]
+];
 const defaultFormat = process.env.SCREENIE_DEFAULT_FORMAT || 'jpeg';
 
+/*
+ * Clean up the PhantomJS pool before exiting when receiving a termination
+ * signal.
```

Comment:
Maybe add a brief note about the magic 143 exit status code. I happened to come across the documentation which states that default SIGTERM behavior is to exit with 128 + signal number, and TERM has signal number 15. But that's definitely not intuitive from simply reading this code :)